sqldb/migrations_dev.go

```diff
@@ -8,4 +8,13 @@ var migrationAdditions = []MigrationConfig{
 		Version:       8,
 		SchemaVersion: 7,
 	},
+	{
+		Name:          "kv_graph_migration",
+		Version:       9,
+		SchemaVersion: 7,
```

Comment:
Hmm, not sure how the `SchemaVersion` currently works. Since the graph migration above has version 8 I kind of would expect this to require version 8 too? cc @bhandras.

Reply:
So this just means that the custom migration will migrate the schema up to version 7 before we run it, while version is our global version which includes all migrations, schema and in-code as well.